main.go

```diff
@@ -104,44 +100,50 @@ func main() {
 		Client:           mgr.GetClient(),
 		Log:              ctrl.Log.WithName("controllers").WithName("ManagedOCS"),
 		Scheme:           mgr.GetScheme(),
-		ConfigSecretName: envVars.configName,
+		ConfigSecretName: EnvVars[addonParamsSecretEnvVarName],
 	}).SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "ManagedOCS")
 		os.Exit(1)
 	}
 	// +kubebuilder:scaffold:builder
 
-	if err := ensureManagedOCS(mgr.GetClient(), setupLog, namespace); err != nil {
+	if err := ensureManagedOCS(mgr.GetClient(), setupLog, EnvVars[namespaceEnvVarName]); err != nil {
 		os.Exit(1)
 	}
 
-	setupLog.Info("starting manager")
+	setupLog.Info("Starting manager")
 	if err := mgr.Start(ctrl.SetupSignalHandler()); err != nil {
-		setupLog.Error(err, "problem running manager")
+		setupLog.Error(err, "Problem running manager")
 		os.Exit(1)
 	}
 }
 
-// getWatchNamespace returns the Namespace the operator should be watching for changes
+/* // getWatchNamespace returns the Namespace the operator should be watching for changes
 func getNamespace() (string, error) {
 
-	ns, found := os.LookupEnv(namespaceEnvVar)
+	ns, found := os.LookupEnv(namespaceEnvVarName)
 	if !found {
-		err := fmt.Errorf("%s must be set", namespaceEnvVar)
+		err := fmt.Errorf("%s must be set", namespaceEnvVarName)
 		return "", err
 	}
 	return ns, nil
 }
-
+*/
 // The name of the secret containing the addon params
-func getEnvVars(envVars *EnvVars) error {
+func getEnvVars(m map[string]string) error {
```

Comment:
Why do we need to provide the map from the outside? can we just instantiate it here and return it?